README.md

```diff
@@ -1,179 +1,169 @@
+Further details regarding the usage of NetCoreDbg can be found in the [CLI](docs/cli.md) manual.
 
+## Building from Source Code
 
-## Building from source code
-
-Currently Netcoredbg can be built on Linux, MacOS or Windows. Instructions for building Netcoredbg on each platform is shown below.
-
+Currently, NetCoreDbg can be built on Linux, MacOS, or Windows. Instructions for building NetCoreDbg on each platform is shown below.
 
 ### Unix
 
-Building of Netcoredbg requires Microsoft's .NET, so currently you can build Netcoredbg only in Linux. Microsoft supports at least few distributions, see details here: https://docs.microsoft.com/en-us/dotnet/core/install/linux
+NetCoreDbg's build requires Microsoft's .NET framework, and as such, NetCoreDbg can only be built in Linux. Microsoft supports a few distributions, the details of which can be found here: https://docs.microsoft.com/en-us/dotnet/core/install/linux.
 
 #### Prerequisites
 
-1. You need to install `cmake`, and `make` or `ninja`.
+1. You need to install `cmake`, and either `make` or `ninja`.
 
-2. You need clang C++ compiler installed (Netcoredbg can't be built with gcc).
+2. You need the clang C++ compiler installed (NetCoreDbg can't be built with gcc).
 
-3. Microsoft's **.NET runtime** should be installed, you can download it here: https://dotnet.microsoft.com/download
+3. Microsoft's **.NET runtime** should be installed, which you can download here: https://dotnet.microsoft.com/download.
 
-4. May be you need to install some typical developers tools not mentioned here, like `git`, etc...
+4. You may also need to install some common developers tools not mentioned here, such as [Git](https://www.git-scm.com/downloads), etc...
 
-5. It is expected, that Netcoredbg sources placed to some directory;
+5. It is expected that you place the NetCoreDbg sources within a directory.
 
-6. Optional step: Netcoredbg requires **CoreCLR runtime source code**, which is typically downloaded automatically, but you can download it from here: https://github.com/dotnet/runtime
+6. Optional step: NetCoreDbg requires the **CoreCLR runtime source code**, which is typically downloaded automatically, but can also be downloaded manually from here: https://github.com/dotnet/runtime.
 
-   For example, you can check out tag v8.x.
+   *For example, you can check out tag v8.x.*
 
-7. Optional step: Netcoredbg requires **.NET SDK**, which can be downloaded automatically, but you can download it manually from here: https://dotnet.microsoft.com/download
+7. Optional step: NetCoreDbg requires the **.NET SDK**, which is typically downloaded automatically, but can also be downloaded manually from here: https://dotnet.microsoft.com/download.
 
 #### Compiling
 
-Configure build with the following commands:
+Configure the build with the following commands:
 
 ```
-user@netcoredbg$ mkdir build
-user@netcoredbg$ cd build
+user@NetCoreDbg$ mkdir build
```

Comment:
I don't think this kind of changes needed, `netcoredbg` is a default name of dir after clone